app/containers/MassEnergizeSuperAdmin/Gallery/tools/LightAutoComplete.js

```diff
@@ -349,31 +350,30 @@ function LightAutoComplete(props) {
                 </div>
               )}
 
-              {(query ? filteredItems : optionsToDisplay).map((op, index) => {
-                return (
-                  <div
-                    key={index?.toString()}
-                    className={classes.dropdownItem}
-                    onClick={() => handleSelection(op)}
-                    style={{
-                      display: "flex",
-                      flexDirection: "row",
-                      alignItems: "center"
-                    }}
-                  >
-                    {multiple && (
-                      <Checkbox
-                        style={{
-                          padding: 0,
-                          marginRight: 6
-                        }}
-                        checked={onlyValues.includes(getValue(op))}
-                      />
-                    )}
-                    {getLabel(op)}
-                  </div>
-                );
-              })}
+            {(filterFunc ? filterFunc(query ? filteredItems : optionsToDisplay) : (query ? filteredItems : optionsToDisplay))
```

Comment:
Eeei, please pull this into it's own flow (in a function)... It's getting tooo crowded